helix-core/src/main/java/org/apache/helix/manager/zk/client/HelixZkClient.java

```diff
@@ -172,6 +186,15 @@ void asyncSetData(final String path, Object datat, final int version,
 
   long getSessionId();
 
+  /**
+   * Gets the zookeeper's session id of this zk client in hexadecimal notation.
+   * This session could be valid, expired, or "0"(initial value before zk session is established).
+   * Ex. 1000a5ceb930004 is returned.
+   *
+   * @return String representation of session id in hexadecimal notation.
+   */
+  String getHexSessionId();
```

Comment:
FYI, I think we want to expose as little information as possible from this interface. We leave getSessionId mainly for backward compatibility. If there is no critical usage, please do not expose more internal session ID (which is changeable) to the ZkClient user.